frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -0,0 +1,463 @@
+      await Model.ensureSavedState();
+      downloadAnnotation(annotationId, annotationType, hasVolumeFallback, {}, includeVolumeData);
+      onClose();
+    } else if (activeTabKey === "export") {
+      const missingSelection = selectedLayerName === "" || selectedBoundingBoxID === -1;
+      const basicWarning = "Starting an export job with the chosen parameters was not possible.";
+      const missingSelectionWarning = " Please choose a layer and a bounding box for export.";
+
+      if (missingSelection) {
+        Toast.warning(basicWarning + missingSelectionWarning);
+      } else {
+        const selectedLayer = getLayerByName(dataset, selectedLayerName);
+        const selectedBoundingBox = userBoundingBoxes.find(
+          (bbox) => bbox.id === selectedBoundingBoxID,
+        );
+        if (selectedLayer != null && selectedBoundingBox != null) {
+          const layerInfos = getLayerInfos(
+            selectedLayer,
+            tracing,
+            activeMappingInfos,
+            isMergerModeEnabled,
+          );
+          await handleStartExport(
+            dataset,
+            layerInfos,
+            selectedBoundingBox.boundingBox,
+            startedExports,
+            setStartedExports,
+          );
+          Toast.success("A new export job was started successfully.");
+        } else {
+          Toast.warning(basicWarning);
+        }
+      }
+      if (!keepWindowOpen && !missingSelection) {
+        onClose();
+      }
+    }
+  };
+
+  const maybeShowWarning = () => {
+    if (activeTabKey === "download" && hasVolumeFallback) {
+      return (
+        <Row>
+          <Text
+            style={{
+              margin: "6px 12px",
+            }}
+            type="warning"
+          >
+            {messages["annotation.no_fallback_data_included"]}
+          </Text>
+        </Row>
+      );
+    } else if (activeTabKey === "python") {
+      return (
+        <Row>
+          <Text
+            style={{
+              margin: "6px 12px",
+            }}
+            type="warning"
+          >
+            {messages["annotation.python_do_not_share"]}
+          </Text>
+        </Row>
+      );
+    }
+    return null;
+  };
+
+  const handleTabChange = (key: string) => {
+    setActiveTabKey(key);
+  };
+
+  const handleCheckboxChange = (checkedValues: CheckboxValueType[]) => {
+    setIncludeVolumeData(checkedValues.includes("Volume"));
+  };
+
+  const handleKeepWindowOpenChecked = (e: any) => {
+    setKeepWindowOpen(e.target.checked);
+  };
+
+  const workerInfo = (
+    <Row>
+      <Divider
+        style={{
+          margin: "18px 0",
+        }}
+      />
+      <Text
+        style={{
+          margin: "6px 12px",
+        }}
+        type="warning"
+      >
+        {messages["annotation.export_no_worker"]}
+        <a href="mailto:hello@webknossos.com">hello@webknossos.com.</a>
+      </Text>
+    </Row>
+  );
+
+  const checkboxStyle = {
+    height: "30px",
+    lineHeight: "30px",
```

Comment:
```suggestion
    height: 30,
    lineHeight: 30,
```

Reply:
I've left this as is, since it leads to the Checkbox suddenly spacing weirdly.
![Screenshot 2022-05-18 at 15 57 28](https://user-images.githubusercontent.com/46814136/169058981-262745e4-d945-4320-b9e5-7a54dc3da7c7.png)